File: app/src/main/java/br/com/wcisang/alojinha/service/response/BannerResponse.java
package br.com.wcisang.alojinha.service.response;

import java.util.List;

import br.com.wcisang.alojinha.model.Banner;

/**
 * Created by WCisang on 07/06/2018.
 */
public class BannerResponse {

    List<Banner> data;

    public List<Banner> getData() {
        return data;
    }

    public void setData(List<Banner> data) {
        this.data = data;
    }
}

File: app/src/main/java/br/com/andrecouto/alodjinha/data/source/remote/service/response/BannerResponse.kt
package br.com.andrecouto.alodjinha.data.source.remote.service.response

import br.com.andrecouto.alodjinha.domain.model.lodjinha.Banner
import com.google.gson.annotations.SerializedName

data class BannerResponse (@SerializedName("data") var data : List<Banner>)
